docs/source/release_notes/release-notes.rst

```diff
@@ -89,6 +89,19 @@ develop
          - Fixed a bug where ``AwaitingLeadershipProxy`` stops trying to gain leadership, causing client calls to leader to throw ``NotCurrentLeaderException``.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3582>`__)
 
+    *    - |userbreak|
+         - As part of preparatory work to migrate to a new transactions table, this version of AtlasDB and all versions going forward expect to be using a version of TimeLock that supports the ``startIdentifiedAtlasDbTransaction`` endpoint.
+           Support for previous versions of TimeLock has been dropped; please update your TimeLock server.
```

Comment:
is there a transitive dependency on the timelock server that we can add?

Reply:
sls-wise, that is

Reply:
(discussed offline - this can be done internally through a plugin, though we still rely on users being up-to-date with that plugin)

Reply:
Should we consider https://github.com/palantir/atlasdb/blob/cdeb25712e51490fe9b0cc9b0497cf383660b8ce/atlasdb-cassandra/build.gradle#L86?

Reply:
Ah, good call. I wasn't aware we _declared_ these in OSS, will add.